<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Oasis Pro Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="sldrBattery">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="Line" name="lDivisor">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>16</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblControlTitle">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Control the Device</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblBatt">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblBattLB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblBattUB">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>291</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gButtons">
     <item row="2" column="1">
      <widget class="QPushButton" name="btnUp">
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnLeft">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnDown">
       <property name="text">
        <string>v</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="vp1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnSelect">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btnPower">
       <property name="text">
        <string>â­˜</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnRight">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnHist">
       <property name="text">
        <string>HISTORY</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="vp2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnRec">
       <property name="text">
        <string>RECORD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnRet">
       <property name="text">
        <string>MAIN</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OASIS PRO SIMULATOR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="lControl">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="screenLayout">
     <item>
      <widget class="QTextBrowser" name="statusBar">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>31</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;CONN: NONE | BATT: 100%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="screen">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>191</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pOff">
        <widget class="QTextBrowser" name="offScreen">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>291</width>
           <height>191</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Power Off&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pUser">
        <widget class="QListWidget" name="userScreen">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>291</width>
           <height>191</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>User 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Add User</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="pMain">
        <widget class="QListWidget" name="mainScreen">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>291</width>
           <height>191</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>191</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Start Session</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Treatment History</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Change User</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="pGroup">
        <widget class="QListWidget" name="groupScreen">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>291</width>
           <height>191</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>20 minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>45 minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>User Designated</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="pType">
        <widget class="QListWidget" name="typeScreen">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>291</width>
           <height>191</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>MET</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Delta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Theta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Alpha</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="pActvSess">
        <widget class="QSlider" name="sldrIntLeft">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>18</width>
           <height>160</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
        <widget class="QLabel" name="lblIntTitle">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>80</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Intensity:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSlider" name="sldrIntRight">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>10</y>
           <width>18</width>
           <height>160</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
        <widget class="QLabel" name="lblInt1">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>150</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lblInt8">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>10</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lblInt7">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lblInt6">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>50</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lblInt5">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lblInt4">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>90</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lblInt3">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>110</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lblInt2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>130</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QListWidget" name="activeSessScreen">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>291</width>
           <height>191</height>
          </rect>
         </property>
        </widget>
        <zorder>activeSessScreen</zorder>
        <zorder>sldrIntLeft</zorder>
        <zorder>lblIntTitle</zorder>
        <zorder>sldrIntRight</zorder>
        <zorder>lblInt1</zorder>
        <zorder>lblInt8</zorder>
        <zorder>lblInt7</zorder>
        <zorder>lblInt6</zorder>
        <zorder>lblInt5</zorder>
        <zorder>lblInt4</zorder>
        <zorder>lblInt3</zorder>
        <zorder>lblInt2</zorder>
       </widget>
       <widget class="QWidget" name="pHist">
        <widget class="QListWidget" name="histScreen">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>291</width>
           <height>191</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="cboConn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Okay</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Excellent</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblConn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection to Skin: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
